pkgs/development/python-modules/pysubs2/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pythonImportsCheckHook
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "pysubs2";
+  version = "1.6.1";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "tkarabela";
+    repo = pname;
+    rev =  version;
+    hash = "sha256-0bW9aB6ERRQK3psqeU0Siyi/8drEGisAp8UtTfOKlp0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pythonImportsCheckHook
+  ];
+
```

```suggestion
  ];

  pythonImportsCheck = [
    "pysub"
  ];
```
Thanks for adding pythonImportsCheck.
`pythonImportsCheckHook` is already included in nativeBuildInputs. 
https://github.com/NixOS/nixpkgs/blob/d37f585a546c35cf800840c96f690fdb351a7a91/pkgs/development/interpreters/python/mk-python-derivation.nix#L195-L197
So there is no need to add it here.
Use it like this.

---

Ah, should have read the docs more carefully. My bad :)